lib/config.js

```diff
@@ -1,33 +1,21 @@
 /*
  * Bedrock redis configuration.
  *
- * Copyright (c) 2012-2018 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
  */
-const {config} = require('bedrock');
+import {config} from '@bedrock/core';
 
 config.redis = {};
 config.redis.host = 'localhost';
 config.redis.port = 6379;
-
-// A string used to prefix all used keys (e.g. namespace:test).
-// `prefix` is useful when multiple bedrock applications share the same redis
-// instance in order to prevent key collisions and simplify debugging.
-// `prefix` should be as few characters as possible because the memory
-// footprint of every key will be increased by one byte per character.
-config.redis.prefix = null;
-
 // retry backoff control
-config.redis.retry = {
+config.redis.retryOptions = {
```

Comment:
Resolved here 64e5a2ce36c8aeff4ebae2e29f2a46dde74e55f2

The reason it was renamed was because `retry.retries` sounded funky to me. But `retryOptions` just moved the problem to a different place as you mentioned.